flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/stream/sql/validation/MatchRecognizeValidationTest.scala

```diff
@@ -42,4 +44,25 @@ class MatchRecognizeValidationTest extends TableTestBase {
     val sql = "SELECT MATCH_PROCTIME() FROM MyTable"
     streamUtil.verifySql(sql, "n/a")
   }
+
+  /** Python Function can not be used in MATCH_RECOGNIZE for now **/
+  @Test
+  def testMatchPythonFunction() = {
+    expectedException.expectCause(Matchers.isA(classOf[TableException]))
```

Comment:
Because in flink planner, this kind of exceptions thrown in physicalNode initiailization is wrapped in a new RuntimeException with new error message, it is hard to catch the error message what we want.